src/index.js

```diff
@@ -1,51 +1,75 @@
-import program from 'commander';
+import {Program} from './lib/program';
+import {Store} from './lib/config/store';
 import glob from 'glob';
 import logger from 'winston';
-
 import BundleCommand from './commands/bundle';
 import InitCommand from './commands/init';
 import NewCommand from './commands/new';
 
 class Aurelia {
+
   constructor() {
     this.commands = {};
+    this.aliases  = {};
     this.name = 'Aurelia CLI tool';
     this.config = {};
     this.logger = logger;
   }
 
   init(config) {
-    this.config = config;
-    let bundle = new BundleCommand(program, this.config, this.logger);
-    let init = new InitCommand(program, this.config, this.logger);
-    let newCmd = new NewCommand(program, this.config, this.logger);
+    this.program      = new Program(config);
+    this.store        = new Store(config);
+    this.config       = config;
+    this.config.store = this.store;
```

Comment:
@ahmedshuhel 
Added store to config. I don't want to keep it here, I am working on another way of using it.